_includes/components/image-left-text-right.njk

```diff
@@ -0,0 +1,12 @@
+<div class="image-l-text-r-block">
+	<div class="container container--flex container--wide">
+		<div class="image-l-text-r-block__image">
+			<img src="/static/img/NLC_lead_the_way-TRANSPARENTX500 1.png" alt="NLC coil">
```

Comment:
The NLC coil is purely decorative here, so it's actually better for screen readers to not read it out at all. We can do this by setting alt=""

Reply:
Amended :)